packages/pwa-kit-react-sdk/src/ssr/universal/components/with-react-query/index.js

```diff
@@ -43,7 +43,14 @@ export const withReactQuery = (Wrapped) => {
 
             const queryCache = queryClient.getQueryCache()
             const queries = queryCache.getAll().filter((q) => q.options.enabled !== false)
-            await Promise.all(queries.map((q) => q.fetch()))
+            await Promise.all(
+                queries.map((q) =>
+                    // If there's an error in this fetch, react-query will log the error
+                    q.fetch().catch(() => {
+                        // On our end, simply catch any error and move on to the next query
+                    })
+                )
+            )
```

Comment:
Thanks for catching that oversight @vmarta 👍 

The side effect here is that the error state isn't dehydrated as per the default behaviour. As we discussed IRL this means for pages that handle errors we would have an have the behaviour of the error page not being rendered server side, then when the query runs client side it would error out and show the error page. I don't think this is desirable behaviour for our specific SSR flow. 

I looked at the API docs and saw that we can choose to dehydrate all the queries like this:

```
dehydrate(queryClient, {shouldDehydrateQuery: () => true})
```

The result of this setting is that we will dehydrate the errors and send them to the client so on first render they have their error shown. 

On a side note we discovered another issue in that we don't stop queries from executing during hydration. This also causes some weird behaviour. I'll create a ticket for that.